packages/nodes-base/nodes/Microsoft/Outlook/FolderDescription.ts

```diff
@@ -30,18 +30,18 @@ export const folderOperations: INodeProperties[] = [
 				description: 'Get a single folder details',
 				action: 'Get a folder',
 			},
-			{
-				name: 'Get All',
-				value: 'getAll',
-				description: 'Get all folders under the root folder of the signed-in user',
-				action: 'Get all folders',
-			},
 			{
 				name: 'Get Children',
 				value: 'getChildren',
 				description: 'Lists all child folders under the folder',
 				action: 'Get items in a folder',
 			},
+			{
+				name: 'Get Many',
+				value: 'getAll',
+				description: 'Get all folders under the root folder of the signed-in user',
+				action: 'Get all folders',
+			},
```

Comment:
order changed here

Reply:
As discussed, order should take into account the new name, so this is correct.